internal/gapicgen/generator/gapics.go

```diff
@@ -266,14 +266,42 @@ func (g *GapicGenerator) microgen(conf *microgenConfig) error {
 
 // manifestEntry is used for JSON marshaling in manifest.
 type manifestEntry struct {
-	DistributionName  string `json:"distribution_name"`
-	Description       string `json:"description"`
-	Language          string `json:"language"`
-	ClientLibraryType string `json:"client_library_type"`
-	DocsURL           string `json:"docs_url"`
-	ReleaseLevel      string `json:"release_level"`
+	DistributionName  string      `json:"distribution_name"`
+	Description       string      `json:"description"`
+	Language          string      `json:"language"`
+	ClientLibraryType string      `json:"client_library_type"`
```

Comment:
What's the difference between `library_type` and `client_library_type`? Fine if there is a difference, just want to make sure we shouldn't use this field.